Manifests/ManifestsApple/com.apple.system-extension-policy.plist

```diff
@@ -204,6 +204,8 @@
 			<string>Allowed System Extensions</string>
 			<key>pfm_type</key>
 			<string>dictionary</string>
+			<key>pfm_value_import_processor</key>
```

Comment:
Whenever you update a manifest, the pfm_last_modified date at the top of the manifest needs to be updated.  Only when preferences are added / removed do you also then need to bump the pfm_version key.

Reply:
Done